packages/braid-design-system/src/lib/components/TextLink/TextLink.docs.tsx

```diff
@@ -71,6 +80,34 @@ const docs: ComponentDocs = {
           </Text>,
         ),
     },
+    {
+      description: (
+        <Text>
+          By default, an icon will be <Strong>leading</Strong> the label,
+          however this can be changed by providing <Strong>iconPosition</Strong>{' '}
+          of <Strong>trailing</Strong>.
+        </Text>
+      ),
+      Example: () =>
+        source(
+          <Inline space="large" alignY="center">
+            <Text>
+              <TextLink icon={<IconArrow direction="left" />} href="">
+                Back
+              </TextLink>
+            </Text>
+            <Text>
+              <TextLink
+                icon={<IconArrow direction="right" />}
+                iconPosition="trailing"
```

Comment:
Running this in Playroom works, but `iconPosition` doesn't appear in the autocomplete dropdown.

<img width="444" alt="iconPosition" src="https://user-images.githubusercontent.com/3297808/223600312-d327f74a-09c4-4d0a-9a23-96fe330601dd.png">

Reply:
Now that you mention it, `icon` isn't showing up either, both in `Button` and `TextLink`. This is the case in master as well, not specific to this branch.

Reply:
Seems like playroom just isn't seeing those props.
Logged during `playroom start`.
```ts
{
  ButtonTypes: {
    tone: [ 'brandAccent', 'critical', 'neutral' ],
    variant: [ 'transparent', 'solid', 'ghost', 'soft' ],
    keyboardFocusable: [],
    radius: [ 'large', 'full' ],
    forceActive: []
  },
  TextLinkTypes: {
    weight: [ 'regular', 'weak' ],
    showVisited: [],
    hitArea: [ 'large', 'standard' ],
    reset: [
      'symbol',   'object', 'strong',   'body',       'small',
      'text',     'link',   'slot',     'style',      'title',
      'clipPath', 'filter', 'mask',     'marker',     'progress',
      'center',   'ruby',   'table',    'data',       'cite',
      'form',     'label',  'pattern',  'span',       'summary',
      'button',   'meter',  'textarea', 'sub',        'circle',
      'embed',    'pre',    'caption',  'menu',       'menuitem',
      'div',      'a',      'abbr',     'address',    'area',
      'article',  'aside',  'audio',    'b',          'base',
      'bdi',      'bdo',    'big',      'blockquote', 'br',
      'canvas',   'code',   'col',      'colgroup',   'datalist',
      'dd',       'del',    'details',  'dfn',        'dialog',
      'dl',       'dt',     'em',       'fieldset',   'figcaption',
      'figure',   'footer', 'h1',       'h2',         'h3',
      'h4',       'h5',     'h6',       'head',       'header',
      'hgroup',   'hr',     'html',     'i',          'iframe',
      'img',      'input',  'ins',      'kbd',        'keygen',
      'legend',   'li',     'main',     'map',        'mark',
      'meta',     'nav',    'noindex',  'noscript',   'ol',
      'optgroup', 'option', 'output',   'p',          'param',
      ... 76 more items
    ]
  }
}
```

Reply:
`ButtonTypes` seems to match the props for `ButtonOverlays`
https://github.com/seek-oss/braid-design-system/blob/37c6d00ea34a78e068673d33eaa6dbf01187c3e7/packages/braid-design-system/src/lib/components/Button/Button.tsx#L224-L230

Reply:
Ahhh, `react-docgen-typescript` must be picking it preferentially for some reason.

Reply:
[This option](https://github.com/styleguidist/react-docgen-typescript#componentnameresolver) might less us specify that the component we want is the one with the same name as the file. Not ideal though.

Reply:
I have noticed this in the passed with other prop/component combinations, so definitely a good one to look into separately 👏